src/controllers/TasksWithTagsController.tsx

```diff
@@ -1,6 +1,19 @@
 import { supabase } from "../supabase/db";
 import { Database } from "../supabase/supabase";
 
+export const getTagsOfTask = async (task_id: number) => {
```

Comment:
I think I implemented something similar with my filter branch that just got merged. Whatever, we can refactor in the future if necessary.